app/main.py

```diff
@@ -1,3 +1,102 @@
-def shop_trip():
-    # write your code here
-    pass
+from app.customer import Customer
+from app.shop import Shop
+from app.car import Car
+
+
+from math import sqrt
+import json
+import datetime
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as file:
+        data = json.load(file)
+
+    fuel_price = data["FUEL_PRICE"]
+    check_data = datetime.datetime.now().strftime("%d/%m/%Y %H:%M:%S")
+
+    customers_list = [
```

Comment:
customers_list: customers is already mean not one)
customers_list -> customers
The same with shops_list, cars_list

Reply:
do not quite understood